Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -2218,6 +2219,16 @@ def long_running_execution_command(client: Client, params: dict):
     mirror_options = params.get('mirror_options', '')
     if not argToBoolean(params.get('retry_events_fetch', True)):
         EVENTS_SEARCH_TRIES = 1
+    is_reset_triggered()
+    context_data, version = get_integration_context_with_version()
+    last_highest_id_last_run = demisto.getLastRun().get(LAST_FETCH_KEY, 0)
+    print_debug_msg(f'Last highest ID from last run: {last_highest_id_last_run}')
+    last_highest_id_context = context_data.get(LAST_FETCH_KEY, 0)
+    if last_highest_id_last_run != last_highest_id_context:
```

Comment:
for integration that is already running, if you update the integration, then last_highest_id_last_run should be 0 and context_data.get(LAST_FETCH_KEY, 0) should be bigger than 0. 
In this case, last_highest_id_last_run should actually be updated from context.

both of the cases are worth of a unit test.